modular_skyrat/modules/aesthetics/vending/access_vending.dm

```diff
@@ -42,12 +42,12 @@
 
 	var/mob/living/carbon/carbon_user = user
 	var/obj/item/card/id/user_id = carbon_user.get_idcard(TRUE)
-	if(!user_id)
+	if(onstation && !user_id && !(obj_flags & EMAGGED))
```

Comment:
vending machines should not be made free via emags, why was this balance change included in a PR titled "unrestricted some items for ghostcafe"?